app/controllers/import_controller.rb

```diff
@@ -203,82 +194,57 @@ def validate_required_field(field, value, row_ind)
     value
   end
 
-  def validate_enum_field(field, value, row_ind)
+  def import_enum_field(field, value)
     return nil if value.blank?
 
     normalized_value = normalize_enum_field_value(value)
-    return NORMALIZED_ENUMS[field][normalized_value] if NORMALIZED_ENUMS[field].keys.include?(normalized_value)
-
-    err_msg = "'#{value}' is not an acceptable value for '#{VALIDATION[field][:label]}',"\
-              " acceptable values are: #{VALID_PATIENT_ENUMS[field].reject(&:blank?).to_sentence}"
-    raise ValidationError.new(err_msg, row_ind)
+    NORMALIZED_ENUMS[field].keys.include?(normalized_value) ? NORMALIZED_ENUMS[field][normalized_value] : value
   end
 
-  def validate_bool_field(field, value, row_ind)
+  def import_and_validate_bool_field(field, value, row_ind)
     return value if value.blank?
     return (value.to_s.downcase == 'true') if %w[true false].include?(value.to_s.downcase)
 
-    err_msg = "'#{value}' is not an acceptable value for '#{VALIDATION[field][:label]}', acceptable values are: 'True' and 'False'"
+    # NOTE: The controller still validates boolean values, since validating those on the model does not work
+    # because by that point they will have been typecast from a string to a bool
+    err_msg = "Value '#{value}' for '#{VALIDATION[field][:label]}' is not an acceptable value, acceptable values are: 'True' and 'False'"
     raise ValidationError.new(err_msg, row_ind)
   end
 
-  def validate_date_field(field, value, row_ind)
-    return nil if value.blank?
-    return value if value.instance_of?(Date)
-
-    unless value.match(/\d{4}-\d{2}-\d{2}/)
-      err_msg = "'#{value}' is not a valid date for '#{VALIDATION[field][:label]}'"
-      if value.match(%r{\d{2}/\d{2}/\d{4}})
-        raise ValidationError.new("#{err_msg} due to ambiguity between 'MM/DD/YYYY' and 'DD/MM/YYYY', please use the 'YYYY-MM-DD' format instead", row_ind)
-      end
-
-      raise ValidationError.new("#{err_msg}, please use the 'YYYY-MM-DD' format", row_ind)
-    end
-
-    begin
-      Date.parse(value)
-    rescue ArgumentError
-      raise ValidationError.new("'#{value}' is not a valid date for '#{VALIDATION[field][:label]}'", row_ind)
-    end
+  def import_date_field(value)
+    value.blank? ? nil : value
   end
 
-  def validate_phone_field(field, value, row_ind)
-    return nil if value.blank?
-
-    phone = Phonelib.parse(value, 'US')
-    return phone.full_e164 unless phone.national(false).nil? || phone.national(false).length != 10
-
-    raise ValidationError.new("'#{value}' is not a valid phone number for '#{VALIDATION[field][:label]}'", row_ind)
+  def import_phone_field(value)
```

Comment:
can this be simplified to just `Phonelib.parse(value, 'US').full_e164 || value` or is there an edge case I'm missing?

Reply:
No, they aren't equivalent because `''` is truthy. An example case is if `value` is `'a'` then `e_164` will be `''`, so in that case we want to return `value`.